### ISSUE TITLE
Is `param.kofam_dir` not used?

### ISSUE THREAD

**Issue by erikrikarddaniel** [LNUc-EEMiS/metatdenovo#143]
### Description of the bug

I tried to run the pipeline with a different directory name for `kofam_dir` (a symlink called `kofamscan`). This failed by first trying to download (the data was already present) and then after download failing with this message:

```
  tar: can't change directory to 'kofam/': No such file or directory
  gunzip: kofam/profiles.tar.gz: No such file or directory
```

I assume `kofam` is hardcoded in places.

After I changed the param to `kofam` it still starts to download.

**Comment by danilodileo** [comment]
I just tried this parameter and it works for me. I had a path called 'kotest' instead of 'kofam' and it worked

**Comment by erikrikarddaniel** [comment]
Den ons 19 apr. 2023 kl 17:49 skrev Danilo Di Leo ***@***.***
>:

> I just tried this parameter and it works for me. I had a path called
> 'kotest' instead of 'kofam' and it worked
>
Did you have a predownloaded database there?

**Comment by danilodileo** [comment]
Yes I had. I might know the issue and I will try to see if it helps

**Comment by danilodileo** [comment]
fixed with PR #155
